src/main/java/com/example/springtweet/springtweet/exceptionHandler/ResponseHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.example.springtweet.springtweet.exceptionHandler;
+
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@ControllerAdvice
+class HandlerMsg {
```

Comment:
The parent class name should be the same as the file name, i.e., ResponseHandler